ui/app/pages/home/home.component.js

```diff
@@ -27,23 +27,14 @@ export default class Home extends PureComponent {
   }
 
   static propTypes = {
-    activeTab: PropTypes.shape({
```

Comment:
This should be removed from `defaultProps` as well. Also `forceApproveProviderRequestByOrigin`, which was mentioned in a previous comment.

Reply:
Good catch, thanks